bepasty_cli/cli.py

```diff
@@ -30,6 +31,18 @@
 LIFETIME_MAPPING = dict(zip(LIFETIME_CHOICES, LIFETIME_NAMES))
 
 
+class LifetimeParamType(click.ParamType):
+    name = 'lifetime'
+    lifetime_regex = re.compile(r'|'.join(r'^([1-9]\d*?) ?({})$'.format(choice) for choice in LIFETIME_CHOICES))
```

Comment:
while giving "099d" would be a bit strange, we do not need to disallow leading zeros, do we?

so, just use `(\d+)` there?